### PR TITLE
Multiarc build

### PR DESCRIPTION
**Pull request by halim-lee**

### CODE REVIEW

File: scripts/build/build-releases.sh [hunk 1]
```diff
@@ -11,46 +11,19 @@
```
Comment (leochr) [inline scripts/build/build-releases.sh:11]: @halim-lee We can get rid of this script and all logic around 'multiple releases' since we'll only need to build a single release. This was something ported from OLO/RCO (which allowed us to manage their images in Docker Hub and refresh them from time to time). But with the images in icr.io and being referenced using SHA values in CSV, we'll have to do a new release.

Suggest to change build-release.sh to build-operator.sh. Thank you

Comment (halim-lee) [reply scripts/build/build-releases.sh]: Yep removed them and testing the scripts!

**Comment by halim-lee** [comment]
Containerize stage flow:

Screenshot 2023-03-15 at 2 59 38 PM

**Comment by halim-lee** [comment]
Builds and tests passed: ci-pipeline ##932

File: .one-pipeline.yaml [hunk 2]
```diff
@@ -227,46 +223,39 @@ containerize:
...
-      apt-get update
-      apt-get -qq -y install build-essential software-properties-common uidmap
       #  Build images
       export RELEASE_TARGET=$(get_env branch)
       
       if [[ -z $DISABLE_ARTIFACTORY ]]; then
-        DISABLE_ARTIFACTORY="false"
+        export DISABLE_ARTIFACTORY="false"
       fi
       if [[ "$DISABLE_ARTIFACTORY" != "false" ]]; then
```
Comment (leochr) [inline .one-pipeline.yaml:237]: @halim-lee This is unrelated to your change. But it'll be good to get rid of this 'big' if-else check that duplicates the steps for non-artifactory. Instead, have smaller if-checks around artifactory specific steps. I am fine with addressing it via a separate PR.

Comment (halim-lee) [reply .one-pipeline.yaml:237]: I will open a separate PR for that change!

**Review by leochr — APPROVED** [review]
@halim-lee Looks great! Thank you!

**Comment by leochr** [comment]
@davco01a fyi, merged this PR

File: .one-pipeline.yaml (hunk 2 above)
Comment (leochr) [reply .one-pipeline.yaml:237]: Sounds good. 

I also merged https://github.ibm.com/websphere/operators/pull/13, so the branch can be switched to main. Thank you.
